src/components/30-organisms/footer/story.js

```diff
@@ -0,0 +1,48 @@
+/* global document */
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from '@storybook/html';
+import './index';
+import { withMarkdown } from '../../../../.storybook/addons/markdown';
+// import withBodyReset from '../../../../.storybook/addons/reset-body';
+import Readme from './README.md';
+
+storiesOf('Organisms/Footer', module)
+  .addDecorator(withMarkdown(Readme))
+  // .addDecorator(withBodyReset())
```

Comment:
why no bodyReset? this brakes footer as it is a 100% with component

Reply:
Will come again after marc merged his branch

Reply:
Merged.